### PR TITLE
VACMS-18435 Facility locator markers to numbers

### PR DESCRIPTION
**Pull request by randimays**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)

### :warning: TeamSites :warning:
Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No

## Summary
Convert the markers in Facility Locator from letters to numbers. Includes updates to styling to make the markers flexible for different zoom levels / browser font sizes.

Also includes an update to move the distance / mileage below the facility name.

Figma for distance / mileage movement: https://www.figma.com/design/GvNVeG9GAXZaBpny7vNxZP/Facility-Locator?node-id=745-13219&m=dev

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18435
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18434

## Testing done
Tested all Facility Types on desktop and mobile including different browser font sizes.

### Mobile

<details><summary>Medium font size</summary>

#### Vet centers / VA cemeteries

<table>
<tr><th>Vet centers</th><th>VA cemeteries</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 21 52 AM</td><td>Screenshot 2024-09-20 at 11 21 59 AM</td></tr>
</table>

<br/>
<br/>

#### VA benefits / Community pharmacies

<table>
<tr><th>VA benefits</th><th>Community pharmacies</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 22 04 AM</td><td>Screenshot 2024-09-20 at 11 22 12 AM</td></tr>
</table>

<br/>
<br/>

#### Community providers / Emergency care

<table>
<tr><th>Community providers</th><th>Emergency care</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 25 24 AM</td><td>Screenshot 2024-09-20 at 11 25 36 AM</td></tr>
</table>

<br/>
<br/>

#### Urgent care / VA health

<table>
<tr><th>Urgent care</th><th>VA health</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 25 43 AM</td><td>Screenshot 2024-09-20 at 11 25 51 AM</td></tr>
</table>

</details>

<details><summary>Other font sizes (List view)</summary>

<table>
<tr><th>Very large</th><th>Large</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 30 04 AM</td><td>Screenshot 2024-09-20 at 11 29 57 AM</td></tr>
</table>

<table>
<tr><th>Small</th><th>Very small</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 29 50 AM</td><td>Screenshot 2024-09-20 at 11 29 42 AM</td></tr>
</table>

</details>

<details><summary>Other font sizes (Map view)</summary>

<table>
<tr><th>Very large</th><th>Large</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 38 49 AM</td><td>Screenshot 2024-09-20 at 11 38 42 AM</td></tr>
</table>

<table>
<tr><th>Small</th><th>Very small</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 38 33 AM</td><td>Screenshot 2024-09-20 at 11 38 22 AM</td></tr>
</table>

</details>

### Desktop

<details><summary>Medium font size</summary>

#### Vet centers

<table>
<tr><th>Search results</th><th>Clicking on result 1 </th></tr>
<tr><td>Screenshot 2024-09-20 at 11 45 58 AM</td><td>Screenshot 2024-09-20 at 11 46 08 AM</td></tr>
</table>

#### VA cemeteries

<table>
<tr><th>Search results</th><th>Clicking on result 4</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 46 18 AM</td><td>Screenshot 2024-09-20 at 11 46 26 AM</td></tr>
</table>

#### VA benefits 

<table>
<tr><th>Search results</th><th>Clicking on result 7</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 46 39 AM</td><td>Screenshot 2024-09-20 at 11 46 45 AM</td></tr>
</table>

#### Community pharmacies

<table>
<tr><th>Search results</th><th>Clicking on result 13</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 46 57 AM</td><td>Screenshot 2024-09-20 at 11 47 05 AM</td></tr>
</table>

#### Community providers

<table>
<tr><th>Search results</th><th>Clicking on result 12</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 47 23 AM</td><td>Screenshot 2024-09-20 at 11 47 35 AM</td></tr>
</table>

#### Emergency care

<table>
<tr><th>Search results</th><th>Clicking on result 1</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 47 45 AM</td><td>Screenshot 2024-09-20 at 11 47 53 AM</td></tr>
</table>

#### Urgent care

<table>
<tr><th>Search results</th><th>Clicking on result 6</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 48 05 AM</td><td>Screenshot 2024-09-20 at 11 48 12 AM</td></tr>
</table>

#### VA health

<table>
<tr><th>Search results</th><th>Clicking on result 5</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 48 21 AM</td><td>Screenshot 2024-09-20 at 11 48 27 AM</td></tr>
</table>

</details>

<details><summary>Other font sizes</summary>

<table>
<tr><th>Very large</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 53 04 AM</td></tr>
</table>

<table>
<tr><th>Large</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 52 57 AM</td></tr>
</table>

<table>
<tr><th>Small</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 52 48 AM</td></tr>
</table>

<table>
<tr><th>Very small</th></tr>
<tr><td>Screenshot 2024-09-20 at 11 52 41 AM</td></tr>
</table>

</details>

## What areas of the site does it impact?

Facility locator only.

## Acceptance criteria
- [x] Within the map and the results listing, numbers are used as the markers to indicate locations
- [x] The markers resize properly according to browser/device font size settings
- [x] Changes made to both mobile and desktop
- [x] For each results listing on the Facility locator, the milage is below the heading/facility name - as shown in the Figma file
- [x] Design review
- [x] Accessibility review 

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed (by @randimays)

### CODE REVIEW

File: src/applications/facility-locator/components/markers/FacilityMarker.jsx [hunk 1]
```diff
@@ -1,18 +0,0 @@
-import DivMarker from './DivMarker';
```
Comment (randimays) [inline src/applications/facility-locator/components/markers/FacilityMarker.jsx:1]: This file is not used or referenced anywhere. It was added to the repo ~4 years ago and the file that referenced it was deleted shortly after.

**Review by powellkerry — APPROVED** [review]

**Review by eselkin — APPROVED** [review]
I'm fine with it, but if they change base font size, the marker will change. Does that potentially lead to problems later? Like markers taking too much space? Does text zoom change the marker size a lot (is it the same marker as on the map)? Waiting on RI because I have something building locally.

**Review by chriskim2311 — APPROVED** [review]
LGTM

**Comment by randimays** [comment]
@eselkin I have some screenshots in the description that show the markers at "very large" text size in Chrome. They do get pretty big, but they're not so large that they cause problems for the other parts of the UI, IMO.

**Comment by laflannery** [comment]
@eselkin and @randimays I'm still waiting for the RI as well but FWIW, before I wrote up this ticket and the testing steps I specifically looked at the sizing of the markers and if I wanted them to scale or not. I ended up at the same conclusion as Randi and thought the benefits of the resizing (ability to see them better, tap them better, etc. - which are the reasons a user might be increasing their font size to begin with) outweighed the possible con of taking up more space on the map.

**Comment by randimays** [comment]
Sorry all; the review instance was working when I requested your reviews. Seems it just died. Same thing happened on Chris's PR as well. Looking into it...

**Comment by randimays** [comment]
@laflannery @eselkin I still can't get the RI for this PR to work. But! I have this commit included in [another Facility Locator PR](https://github.com/department-of-veterans-affairs/vets-website/pull/32031), and its review instance is working: 

http://af26f390210ec3fc693f3447480e41fe.review.vetsgov-internal/find-locations

If you want to test in that review instance, here are the tickets you can test in it:

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18435
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18434
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19095

**Review by laflannery — APPROVED** [review]
Approving for all 3 tickets mentioned in [this comment](https://github.com/department-of-veterans-affairs/vets-website/pull/32021#issuecomment-2364525137)

**Comment by randimays** [comment]
@thejordanwood Forgot to tag you in [this comment](https://github.com/department-of-veterans-affairs/vets-website/pull/32021#issuecomment-2364525137)

**Review by thejordanwood — APPROVED** [review]
Approving the milage and the number tickets! I don't really need to approve the semantic tag ticket.

**Pull request merged**